mathesar/filters.py

```diff
@@ -1,12 +1,21 @@
 from django_filters import rest_framework as filters
 
-from mathesar.models import Table
+from mathesar.models import Schema, Table
 
 
 class CharInFilter(filters.BaseInFilter, filters.CharFilter):
     pass
 
 
+class SchemaFilter(filters.FilterSet):
+    name = CharInFilter(field_name='name', lookup_expr='in')
+    database = CharInFilter(field_name='database', lookup_expr='in')
```

Comment:
`database` should be an exact match, not a "contains".

Reply:
Just to be clear, we want to only filter by a single database and not multiple?

Reply:
Yes, just a single database.

Reply:
Actually, we can support multiple, but they should be exact matches.

Reply:
I believe it currently only makes exact matches. Testable by adding or subtracting from the database names in the `test_schema_list_filter` function.

Reply:
I misunderstood what `In`meant in the `CharInFilter`, sorry.